packages/cubejs-snowflake-driver/driver/SnowflakeDriver.js

```diff
@@ -67,6 +69,17 @@ class SnowflakeDriver extends BaseDriver {
         WHERE columns.table_schema NOT IN ('INFORMATION_SCHEMA')
      `;
   }
+
+  async release() {
+    this.initialConnectPromise.then((connection) => connection.destroy(function(err, conn) {
```

Comment:
I'd suggest to promisify destroy method and `reject()` it on errors and `resolve()` in case of success. Release operation should fail on errors basically.

Reply:
Done! Thanks for pointing this out!